src/Mailer/Message.php

```diff
@@ -1621,7 +1621,7 @@ protected function wrap(?string $message = null, int $wrapLength = self::LINE_LE
         $cut = ($wrapLength === static::LINE_LENGTH_MUST);
 
         foreach ($lines as $line) {
-            if (empty($line) && $line !== '0') {
+            if (!$line && $line !== '0') {
```

Comment:
```suggestion
            if ($line === '') {
```

Reply:
Here theoretically no type is protecting the input. So this could change behavior if people entered some falsey values that are not empty string.